frontend/src/components/Checklists.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { Fragment, useContext } from 'react';
+import {
+  Typography,
+  List,
+  ListItem,
+  Box,
+  Chip,
+  Checkbox,
+} from '@material-ui/core';
+import { Module } from '../types';
+import { AppContext } from '../context';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+import PanoramaFishEyeIcon from '@material-ui/icons/PanoramaFishEye';
+import { useStyles, } from '../styles';
+import ReactMarkdown from 'react-markdown';
+import {
+  getSupportedTools,
+  getSelectedTools,
+} from '../utils/moduleHelpers';
+
+interface Props {
+  module: Module;
+  readOnlyMode: boolean;
```

Comment:
When would we use read-only mode? Is it for viewing checklists that have already been filled out?

Reply:
read only mode is for when we are selecting the modules in the assessment and click on the dropdown to find out more about the checklist items in that module. So we dont want these "tickable".